apps/my-time/my-time-api/dev.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
```

Comment:
you shouldn't deploy a helm release to dev normally, pull requests will deploy an instance per pull request and we don't use a permanent instance there